CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ project(Proteus
 set(CMAKE_CXX_STANDARD 17 CACHE STRING "")
 set(CMAKE_CXX_STANDARD_REQUIRED YES)
 set(CMAKE_CXX_EXTENSIONS NO)
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Create an `ENABLE_PIC` cmake option to optionally set it

Reply:
Done.